src/main/java/org/atlasapi/query/content/merge/FieldMergeStrategy.java

```diff
@@ -4,31 +4,45 @@
 import java.util.Set;
 import java.util.stream.StreamSupport;
 
+import javax.validation.constraints.NotNull;
+
 import org.atlasapi.media.entity.simple.MergeableFields;
 
 import com.google.common.base.Splitter;
 import com.google.common.base.Strings;
+import com.google.common.collect.ImmutableSet;
 
 public class FieldMergeStrategy {
     private static final Splitter COMMA_SPLITTER = Splitter.on(',');
+    private static final Set<MergeableFields> DO_NOT_MERGE_BY_DEFAULT = ImmutableSet.of(
+            MergeableFields.LOCATIONS)
+            ;
 
-    private final boolean mergeEverything;
-    private final Set<MergeableFields> fieldsToMerge = new HashSet<>();
+    private final Set<MergeableFields> fieldsToMerge;
 
-    private FieldMergeStrategy(boolean mergeEverything) {
-        this.mergeEverything = mergeEverything;
+    private FieldMergeStrategy() {
+        this.fieldsToMerge = new HashSet<>();
+    }
+    private FieldMergeStrategy(@NotNull Set<MergeableFields> fieldsToMerge) {
+        this.fieldsToMerge = fieldsToMerge;
     }
 
-    public static FieldMergeStrategy mergeAll() {
-        return new FieldMergeStrategy(true);
+    public static FieldMergeStrategy defaultMergeStrategy() {
+        Set<MergeableFields> fieldsToMerge = new HashSet<>();
+        for (MergeableFields field : MergeableFields.values()) {
+            if (!DO_NOT_MERGE_BY_DEFAULT.contains(field)) {
+                fieldsToMerge.add(field);
+            }
+        }
+        return new FieldMergeStrategy(fieldsToMerge);
     }
 
     public static FieldMergeStrategy create() {
```

Comment:
```suggestion
    public static FieldMergeStrategy none() {
```
If combined with the comment below about making `addFieldToMerge` we can then give this method a more meaningful name, which will improve the readability of the code for there places where we're intentionally doing no merging.